jenkins/1.11/README.md

```diff
@@ -0,0 +1,86 @@
+# How to use  Jenkins on DC/OS
+
+[Jenkins](https://jenkins-ci.org/) is a popular Continuous Integration (CI) automation server and framework with hundreds of plugins (GitHub, Docker, Slack, etc.) available. Running Jenkins on DC/OS allows you to scale your CI infrastructure by dynamically creating and destroying Jenkins agents as demand increases or decreases, and enables you to avoid the statically partitioned infrastructure typical of a traditional Jenkins deployment.
+
+- Estimated time for completion: Approximately 5 minutes
+- Target audience: Anyone who wants to deploy Jenkins and use it to run automatable jobs
+- Scope: You'll learn how to install Jenkins and how to use it
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install Jenkins](#install-jenkins)
+- [Use Jenkins](#use-jenkins)
+- [Uninstall Jenkins](#uninstall-jenkins)
+
+## Prerequisites
+
+- A running DC/OS 1.11 cluster with at least 2 nodes with 2 CPUs and 3GB of RAM available.
+- [DC/OS CLI](https://dcos.io/docs/1.11/usage/cli/install/) installed.
+
+## Install Jenkins
+
+Jenkins stores information directly on disk, therefore we have two options for deployment. We can pin it to a single node, or use a network file system to share files between multiple nodes. In this guide, we will pin Jenkins to a single node. To do so, create the file `options.json` with the configuration below, modifying `pinned-hostname` to correspond to an agent IP in your DC/OS cluster (for example, via `dcos node`):
+
+```bash
+$ cat options.json
+{
+    "storage": {
+        "pinned-hostname": "10.0.3.230"
+    }
+}
+```
+
+Note that for a complete list of the configuration options available for the Jenkins package, see the [Jenkins package definition](https://github.com/mesosphere/universe/tree/version-3.x/repo/packages/J/jenkins) in the Mesosphere Universe.
+
+Once you've created `options.json`, you can then install Jenkins by running the following command:
+
+```bash
+$ dcos package install jenkins --options=options.json
+By Deploying, you agree to the Terms and Conditions https://mesosphere.com/catalog-terms-conditions/#certified-services
+WARNING: If you didn't provide a value for `storage.host-volume` (either using the CLI or via the Advanced Install dialog),
+YOUR DATA WILL NOT BE SAVED IN ANY WAY.
+
+Continue installing? [yes/no] yes
+Installing Marathon app for package [jenkins] version [3.5.0-2.107.2]
+Jenkins has been installed.
+```
+
+Once ready, Jenkins will appear as a service in the DC/OS dashboard.
```

Comment:
Since this can also be done from the Catalog in the web UI without creating an options.json, I might offer the web UI way first, and the CLI way as another option.